frontend/src/components/Reports/routine/activityReportByUnit.js

```diff
@@ -0,0 +1,148 @@
+import React, { useEffect, useState, useRef } from 'react';
+import { FormattedMessage, useIntl } from 'react-intl';
+import { Form, FormLabel, Grid, Column, Section, Button, Loading } from "@carbon/react";
+import CustomDatePicker from "../../common/CustomDatePicker";
+import { AlertDialog } from "../../common/CustomNotification";
+import config from "../../../config.json";
+import TestSelectForm from "../../workplan/TestSelectForm";
+import TestSectionSelectForm from "../../workplan/TestSectionSelectForm";
+import "../../Style.css";
+import { getFromOpenElisServer } from "../../utils/Utils";
+
+const activityReportByUnit = () => {
+  const intl = useIntl();
+  const mounted = useRef(false);
+  const [loading, setLoading] = useState(false);
+  const [notificationVisible, setNotificationVisible] = useState(false);
+  const [SelectedValue, setSelectedValue] = useState(false);
+  const [reportFormValues, setReportFormValues] = useState({
+    startDate: null,
+    endDate: null
+  });
+  const [unitList, setUnitList] = useState([]);
+  
+  const encodeDate = (dateString) => {
+    if (typeof dateString === "string" && dateString.trim() !== "") {
+      return dateString.split("/").map(encodeURIComponent).join("%2F");
+    } else {
+      return "";
+    }
+  };
+
+  const handleChangeDatePicker = (datePicker, date) => {
+    const updatedDate = encodeDate(date);
+    setReportFormValues(prevState => ({
+      ...prevState,
+      [datePicker]: updatedDate
+    }));
+  };
+
+  const handleSubmit = () => {
+    setLoading(true);
+    const baseParams = 'report=activityReportByPanel&type=indicator';
```

Comment:
THis is a wrongParam for the activityReport by unit .
its r`eport=activityReportByTestSection`